index.html

```diff
@@ -24,11 +28,33 @@
           <a href="#">
             <img src="assets/img/top-nav-geometry.svg" alt="geometry design" />
           </a>
-          <img
-            class="hamburger"
-            src="assets/img/ic_menu.svg"
-            alt="hamburger menu"
-          />
+          <div class="hamburger" id="hamburger-icon">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+          <div class="dropdown-menu visibility" id="dropdown-menu">
+            <div class="pop-up-menu">
+              <img src="assets/img/pop-up-top-geo.svg" alt="geometry design" />
+              <div class="cancel cancel-menu">
+                <div class="cancel-bar-1"></div>
+                <div class="cancel-bar-2"></div>
+              </div>
+            </div>
+
+            <div class="mobile-nav-links">
+              <ul>
+                <li>
+                  <a href="#intro-headline" class="cancel-menu">Hello</a>
+                </li>
+                <li>
+                  <a href="#projects-section" class="cancel-menu">Portfolio</a>
+                </li>
+                <li><a href="#about-section" class="cancel-menu">About</a></li>
+                <li><a href="#contact-me" class="cancel-menu">Contact</a></li>
+              </ul>
+            </div>
```

Comment:
- [ ] Kindly make sure the menu follows the design from [your chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g. all mobile menu options must be centered,...).

| Figma design | Your Design |
| ------------------- | ---------------- |
| ![Screenshot from 2022-09-06 02-03-15](https://user-images.githubusercontent.com/83042742/188522380-88eba7a0-a5a7-430e-9e6e-d93576b3bd62.png) | ![Screenshot from 2022-09-06 02-06-53](https://user-images.githubusercontent.com/83042742/188522392-e82a0aa2-7a95-4e81-b461-319f7b5df467.png) |